codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AwsDependency.java

```diff
@@ -52,8 +52,7 @@
     BODY_CHECKSUM_GENERATOR_BROWSER(NORMAL_DEPENDENCY, "@aws-sdk/body-checksum-browser", "^1.0.0-beta.0"),
     BODY_CHECKSUM_GENERATOR_NODE(NORMAL_DEPENDENCY, "@aws-sdk/body-checksum-node", "^1.0.0-beta.0"),
     XML_BUILDER(NORMAL_DEPENDENCY, "@aws-sdk/xml-builder", "^1.0.0-beta.1"),
-    XML_PARSER(NORMAL_DEPENDENCY, "fast-xml-parser", "^3.16.0"),
-    UUID_GENERATOR(NORMAL_DEPENDENCY, "uuid", "^3.0.0"),
+    XML_PARSER(NORMAL_DEPENDENCY, "fast-xml-parser", "^3.16.0"), UUID_GENERATOR(NORMAL_DEPENDENCY, "uuid", "^3.0.0"),
```

Comment:
newline removed by mistake
```suggestion
    XML_PARSER(NORMAL_DEPENDENCY, "fast-xml-parser", "^3.16.0"),
    UUID_GENERATOR(NORMAL_DEPENDENCY, "uuid", "^3.0.0"),
```